src/dashboard/Data/ApiConsole/GraphQLConsole.react.js

```diff
@@ -32,6 +32,7 @@ export default class GraphQLConsole extends Component {
     } else {
       const headers = {
         'X-Parse-Application-Id': applicationId,
+        'X-Parse-Client-Key': clientKey,
```

Comment:
The client key may be not available. Can you please check and only add the header in the case it exists?

Reply:
Done!